src/NServiceBus.Core/Diagnostics/ActivityNames.cs

```diff
@@ -0,0 +1,13 @@
+namespace NServiceBus
+{
+    class ActivityNames
+    {
+        // TODO: do we want to stick with incoming and outgoing message or use the term send instead?
```

Comment:
I'm not really sure what these are used for. We override the display name to be something different anyway. Is it possible to query based on these?